admin-client/src/components/ExternalLink.svelte

```diff
@@ -24,7 +25,7 @@
 <a
   rel="noreferrer"
   target="_blank"
-  class="font-code-2xs padding-x-1 text-primary text-ls-neg-3 text-uppercase"
+  class="font-code-2xs text-primary text-ls-neg-3 text-uppercase {className}"
```

Comment:
does this value get redefined?

Reply:
Yup. Playing with the idea that you can add additional classes to the anchor for customization.  Using it with the '<SiteCard />' to add padding to the center external link.